### ISSUE TITLE
JX Packaging with Sails.js

### ISSUE THREAD

**Issue by tiagodexter** [balderdashy/sails-docs#362]
After i package my app, when I run with JX, its returns an error, but when I run without packaging, I dont have this error.

I'm using these processes:

MacBook-Pro-de-Tiago:~ tiagodexter$ sails new installer
info: Created a new Sails app installer!
MacBook-Pro-de-Tiago:installer tiagodexter$ sudo ../Downloads/jx_osx64/jx package app.js installer
MacBook-Pro-de-Tiago:installer tiagodexter$ sudo ../Downloads/jx_osx64/jx installer.jx

/Users/tiagodexter/installer/node_modules/sails/node_modules/captains-log/node_modules/rc/index.js.jx:2
#! /usr/bin/env node
^
To run an app using node app.js, you usually need to have a version of sails installed in the same directory as your app.
To do that, run npm install sails

Alternatively, if you have sails installed globally (i.e. you did npm install -g sails), you can use sails lift.
When you run sails lift, your app will still use a local ./node_modules/sails dependency if it exists,
but if it doesn't, the app will run with the global sails instead!

And when I run without package, I have this return.

MacBook-Pro-de-Tiago:installer tiagodexter$ sudo ../Downloads/jx_osx64/jx app.js 
info: 
info: 
info: Sails <|
info: v0.10.5 |\
info: /|.\
info: / || \
info: ,' |' \
info: .-'.-==|/--'
info: `--'-------' 
info: _---------------
info: ____----------------__
info: 
info: Server lifted in /Users/tiagodexter/installer
info: To see your app, visit http://localhost:1337
info: To shut down Sails, press + C at any time.

debug: --------------------------------------------------------
debug: :: Mon Dec 08 2014 21:09:39 GMT-0200 (BRST)

debug: Environment : development
debug: Port : 1337
debug: --------------------------------------------------------

Anyone can help me?

**Comment by sailsbot** [comment]
Thanks for posting, @tiagodexter.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
